src/lightning-address.ts

```diff
@@ -1,4 +1,3 @@
-import fetch from 'cross-fetch';
```

Comment:
in the constructor of LightningAddress we could check if global.fetch is defined, if not throw a nice error message

Reply:
globalThis might not even exist, and then the check wouldn't work as far as I know. I think it is easier for users to figure out the issue if they get an error directly from the failed usage of fetch

Reply:
afaik globalThis always exists, doesn't it? and I am not sure if people figure that out.
if you use a library you expect it to work, otherwise you blame the library mostly.

Is a check like this a bad practice?
```
if (!globalThis.fetch) { throw new Error("Fetch is missing. Please install a polyfill"); } 
```

Reply:
I don't think we should do this. We have to add this check everywhere we use fetch. In the alby-js-sdk repo we already have a section where we address this issue - I think is a better solution: https://github.com/getAlby/alby-js-sdk#for-nodejs